notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb

```diff
@@ -0,0 +1,722 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.            gr.Slider(0, 200, value=45, label="Seed"),
</pre><p>is it whole supported range for seed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Seed range set to <code>0, 10000000</code> , taken from another NB </p>